src/foam/nanos/crunch/ui/MinMaxCapabilityWizardlet.js

```diff
@@ -103,6 +103,13 @@ foam.CLASS({
           // TODO: investigate why this is still needed,
           // setting data to empty array should have made isAvailable automatically evaluate to false
           this.choiceWizardlets.forEach(cw => {
+            let alternateFlow = cw.__subContext__.sequence.contextAgentSpecs.filter(x => x.spec.class == "foam.u2.wizard.agents.AlternateFlowAgent");
```

Comment:
```suggestion
            // AlternateFlowAgent can specify FINAL isAvailable settings on wizardlets
            let alternateFlow = cw.__subContext__.sequence.contextAgentSpecs.filter(x => x.spec.class == "foam.u2.wizard.agents.AlternateFlowAgent");
```

Reply:
This should happen outside the for loop

Reply:
Oh nevermind, I guess it shouldn't

Reply:
@KernelDeimos  I agree, we can grab the alternateFlow only once instead of every iteration of the forEach.

Can grab it using the MinMaxCapabilityWizardlet's subContext. I am assuming there shouldn't be multiple AlternateFlowAgents across different wizardlets instantied from the same sequence flow.